auth-api/src/auth_api/utils/user_context.py

```diff
@@ -33,23 +33,14 @@ class UserContext:  # pylint: disable=too-many-instance-attributes
 
     def __init__(self):
         """Return a User Context object."""
-        token_info: Dict = _get_token_info()
+        token_info: Dict = _get_token_info() or {}
         self._token_info = token_info
-        self._user_id = None
-        user_model: UserModel = None
-        if token_info:
-            user_model = UserModel.find_by_jwt_token(token_info)
-
-        if user_model:
-            self._user_id = user_model.id
-        # try to user name from DB ;if not avaiable fall back to token_info
-        self._user_name: str = getattr(user_model, 'username',
-                                       token_info.get('username', token_info.get('preferred_username', None)))
+        self._user_name: str = token_info.get('username', token_info.get('preferred_username', None))
```

Comment:
I think now i understand why u went for username ;so can ignore above pr comments